src/librustc_mir/const_eval.rs

```diff
@@ -41,7 +41,7 @@ const DETECTOR_SNAPSHOT_PERIOD: isize = 256;
 /// that inform us about the generic bounds of the constant. E.g., using an associated constant
 /// of a function's generic parameter will require knowledge about the bounds on the generic
 /// parameter. These bounds are passed to `mk_eval_cx` via the `ParamEnv` argument.
-pub(crate) fn mk_eval_cx<'mir, 'tcx>(
+pub fn mk_eval_cx<'mir, 'tcx>(
```

Comment:
Now it's even more public than before :laughing: I meant that we can make the function private because it's unused outside this module

Reply:
Sorry, this is what happens when I haven't had my coffee yet. 🤦‍♂